### PR TITLE
upgrade to dotnet 6

### PR DESCRIPTION
**Pull request by kelseyduffy**

### CODE REVIEW

**Review by anton-dahlin — APPROVED** [review]

File: OmfIngressClientLibraries/Program.cs [hunk 1]
```diff
@@ -65,7 +65,6 @@ public static void Main()
                 }
 
                 Console.WriteLine("Complete!");
-                Console.ReadKey();
```
Comment (cbardini) [inline OmfIngressClientLibraries/Program.cs:68]: Why is this removed? Does the console no longer close after the program finishes running?

Comment (kelseyduffy) [reply OmfIngressClientLibraries/Program.cs:68]: yea Visual Studio changed the default settings a while ago so that it doesnt close the window after it runs. Also, this was a little confusing with the way it was written because it says Complete! but then waits for a keystroke without telling you it's waiting for a keystroke. Then the process ends, and VS leaves the window open until another keystroke is entered :)

**Review by cbardini — APPROVED** [review]
